static/js/actions/index.js

```diff
@@ -137,6 +145,10 @@ export function fetchDashboard() {
     dispatch(requestDashboard());
     return api.getDashboard().
       then(dashboard => dispatch(receiveDashboardSuccess(dashboard))).
-      catch(() => dispatch(receiveDashboardFailure()));
+      catch(e => {
+        dispatch(receiveDashboardFailure());
+        // propogate exception
+        return Promise.reject(e);
+      });
```

Comment:
ok cool, I was just wondering if it would make sense to surface this to the user in some way - like an 'error: connection troubles' or 'error: not authorized', etc. maybe something to worry about later.